README.md

```diff
@@ -143,6 +144,13 @@ To use Amazon Webservices (AWS) to provide an Elasticsearch service for Moodle:
 1. Create an AWS account: [Account creation guide](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
 2. Setup an Elasticsearch service: [AWS Elasticsearch setup guide](http://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-createupdatedomains.html)
 
+
+## Elastic Cloud Setup
+
+To use Elastic Cloud to provide an Elasticsearch service for Moodle:
+1. Create an Elastic Cloud account and set up Elasticsearch: [Elasticsearch: Getting Started](https://www.elastic.co/webinars/getting-started-elasticsearch?baymax=default&elektra=docs&storm=top-video)
+2. Generate API Key for your search index and set this ket to the  moodle plugin settings.  
```

Comment:
Small typos:
`ket to the  moodle` => `key to the moodle`

Reply:
fixed